install-binary.sh

```diff
@@ -1,23 +1,21 @@
-#!/usr/bin/env bash
+#!/usr/bin/sh sh
 
 # Shamelessly copied from https://github.com/technosophos/helm-template
 
 PROJECT_NAME="helm-diff"
 PROJECT_GH="databus23/$PROJECT_NAME"
-GREP_COLOR="never"
-
-: ${HELM_PLUGIN_DIR:="$(helm home --debug=false)/plugins/helm-diff"}
```

Comment:
We do use HELM_PLUGIN_DIR in `installFile`. Should we just revert this or just make it POSIX-compliant?

Reply:
@mumoshu AFAIK the `:` denotes the line as pretty much a comment and thus would not run. As `helm home` is not supported in helm3

Reply:
@deiga Thanks. I thought so at first, but my tiny experiment revealed that this seems to have actual side-effect of assigning the value.

```
$ bash -c ': ${FOO:=somevalue}; echo $FOO'
somevalue
```

> As helm home is not supported in helm3

Good catch! Then it can be said that this can safely be removed for helm 3. Not yet sure that's alright for helm 2 though.

Reply:
Right. Should I add a conditional for helm2?

Reply:
@deiga That would be awesome! It will allow us to move forward without deciding if it's necessary or not right now.